packages/storybook8/stories/Components/ImageOverlay/utils.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export const controlsToAdd = {
```

Comment:
Let's use the common utils for the arg types and controls, we should not create a separate util for each component. For default value, we can use the new format as this one is deprecated, you can refer to the meta here: https://github.com/Azure/communication-ui-library/pull/4807/files#diff-0247ac45bec4e27310112a2fe95ac172b30e5cd1dc86f4bd3483b68d50b0fadf

Reply:
Resolved using the Controls Utils